source/simulator/helper_functions.cc

```diff
@@ -1457,19 +1454,26 @@ namespace aspect
                 double theta = 1.0;
                 if (std::abs(max_solution_local-local_solution_average) > std::numeric_limits<double>::min())
                   {
-                    theta = std::min(theta, std::abs((max_solution_exact_global-local_solution_average)
-                                                     / (max_solution_local-local_solution_average)));
+                    theta = std::min(theta, (max_solution_exact_global-local_solution_average)
+                                     / (max_solution_local-local_solution_average));
                   }
                 if (std::abs(min_solution_local-local_solution_average) > std::numeric_limits<double>::min())
                   {
-                    theta = std::min(theta, std::abs((min_solution_exact_global-local_solution_average)
-                                                     / (min_solution_local-local_solution_average)));
+                    theta = std::min(theta, (min_solution_exact_global-local_solution_average)
+                                     / (min_solution_local-local_solution_average));
                   }
```

Comment:
Hi, Rene, the theta defined originally using absolute value to make sure it to be a positive number was correct.  For you better understanding the definition, you can check the equation (2.8) at [On maximum-principle-satisfying high order schemes for scalar conservation laws](https://www.math.purdue.edu/~zhan1966/research/paper/maximum_principle.pdf)